chromadb/api/models/Collection.py

```diff
@@ -49,11 +51,15 @@ def __init__(
         name: str,
         id: UUID,
         embedding_function: Optional[EmbeddingFunction] = ef.DefaultEmbeddingFunction(),
+        tenant: Optional[str] = None,
+        database: Optional[str] = None,
         metadata: Optional[CollectionMetadata] = None,
     ):
         super().__init__(name=name, metadata=metadata, id=id)
```

Comment:
I think we need to pass tenant/db into the super constructor for pydantic to track them